app/controllers/providers_controller.rb

```diff
@@ -64,6 +64,21 @@ def publish
   def training_providers
```

Comment:
Not for this PR but I wonder if this is an indication that a separate controller / restful resource is required ... Looks like we are trying to make a distinction between accrediting bodies and accredited bodies

Reply:
Misha and I had a chat whether this should be in a separate controller and decided to keep it here.